Source/WebKit/UIProcess/ios/WKScrollView.mm

```diff
@@ -549,10 +552,31 @@ - (CGPoint)_puic_contentOffsetForCrownInputSequencerOffset:(double)sequencerOffs
 #endif // HAVE(PEPPER_UI_CORE)
 
 #if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
+- (bool)_updateOverlayRegions:(const Vector<CGRect> &)overlayRegions
+{
+    if (_overlayRegions == overlayRegions)
+        return false;
+
+    [self willChangeValueForKey:@"overlayRegions"];
+    _overlayRegions = overlayRegions;
+    [self didChangeValueForKey:@"overlayRegions"];
+    return true;
+}
+
+- (const Vector<CGRect> &)_overlayRegions
+{
+    return _overlayRegions;
+}
+
 - (NSArray<NSData *> *)overlayRegions
 {
-    return [_internalDelegate _overlayRegions];
+    NSMutableArray<NSData *> *overlayRegions = [NSMutableArray array];
```

Comment:
I think this might be more efficient if it used VectorCocoa's `createNSArray(CollectionType&& collection, MapFunctionType&& function)` (if it can)

Reply:
```
    return createNSArray(_overlayRegions, [] (auto& rect) -> NSData * {
        return [NSData dataWithBytes:(void*)&rect length:sizeof(rect)];
    });
```
that should work